cmake/serac-config.cmake.in

```diff
@@ -0,0 +1,103 @@
+# Copyright (c) 2017-2020, Lawrence Livermore National Security, LLC and
+# other serac Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (BSD-3-Clause)
+#------------------------------------------------------------------------------
+
+cmake_minimum_required(VERSION 3.0 FATAL_ERROR)
```

Comment:
BLT requires 3.8.  This also has side effects of loading CMake policies differently depending on what version you list here (even if you are actually using 3.14.0 or whatever).  I would keep it as close to what we actually require.  Not sure why axom chose 3.0 since they definitely dont work with 3.0